website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -9,30 +9,70 @@ description: >-
 
 # Proxy Service Registration
 
-To function as a Connect proxy, proxies must be declared as a proxy types in
-their service definitions, and provide information about the service they
-represent.
+This topic describes how to declare a proxy as a `connect-proxy` in service definitions. The type must be declared and information about the service they represent must be provided to function as a Consul service mesh proxy.
```

Comment:
Might be clearer to say "kind" rather than "type" because that is the name of the field in the service definition
```suggestion
This topic describes how to declare a proxy as a `connect-proxy` in service definitions. The kind must be declared and information about the service they represent must be provided to function as a Consul service mesh proxy.
```